### PR TITLE
ART-2718: When switching to RHSA set the appropriate Impact in the topic

### PR DESCRIPTION
**Pull request by jasinner**
Reworking of https://github.com/openshift/elliott/pull/188 to use existing templating feature of ocp_build_data.

### CODE REVIEW

File: elliottlib/cli/attach_cve_flaws_cli.py [hunk 1]
```diff
@@ -115,6 +116,11 @@ def attach_cve_flaws_cli(runtime, advisory_id, noop, default_advisory_type):
         ))
         advisory.update(security_impact=highest_impact)
 
+    if highest_impact not in advisory.topic:
```
Comment (vfreex) [inline elliottlib/cli/attach_cve_flaws_cli.py:119]: Line 117 only updates the security impact when the current impact of advisory is smaller than `highest_impact`. I think the topic should be updated together with advisory security impact change. Any thought?

**Review by vfreex — CHANGES_REQUESTED** [review]
Thanks for the PR! Generally LGTM, only 1 question. See my comment inline.

**Comment by vfreex** [comment]
Only flake8 needs to be fixed.

File: elliottlib/cli/attach_cve_flaws_cli.py (hunk 1 above)
Comment (jasinner) [reply elliottlib/cli/attach_cve_flaws_cli.py:118]: Agreed, it's better to always set to the highest_impact. I removed the if statement here.

**Comment by jasinner** [comment]
> Only flake8 needs to be fixed.

Fixed now. Thanks.

File: elliottlib/cli/attach_cve_flaws_cli.py [hunk 2]
```diff
@@ -128,7 +133,7 @@ def attach_cve_flaws_cli(runtime, advisory_id, noop, default_advisory_type):
 
     if noop:
         print('DRY-RUN: The following changes would have been applied to the advisory:')
-        print(advisory)
+        print(advisory.dump())
```
Comment (sosiouxme) [inline elliottlib/cli/attach_cve_flaws_cli.py:136]: i observed `advisory.dump()` seems to have a bug:
```
  File "/workspaces/elliott/elliottlib/cli/attach_cve_flaws_cli.py", line 136, in attach_cve_flaws_cli
    print(advisory.dump())
  File "/home/lmeyer/.local/lib/python3.9/site-packages/errata_tool/erratum.py", line 996, in dump
    print("Manager Email: " + self.manager_email)
TypeError: can only concatenate str (not "NoneType") to str
```
`manager_email` should be set there. that's nothing wrong with this code, just something we might want to consider reverting if it causes problems.

**Review by sosiouxme — APPROVED** [review]
LGTM :ship: 

and thanks for the help!

File: elliottlib/cli/attach_cve_flaws_cli.py (hunk 2 above)
Comment (jasinner) [reply elliottlib/cli/attach_cve_flaws_cli.py:136]: Ah yes, I remember hitting from problem now. I think I removed offending line from erratum.py but that means it doesn't print the manager_email. If we don't use dump we don't get the updated description section of the erratum.

**Pull request merged**